deployer/src/algolia_internal_api.py

```diff
@@ -19,6 +19,7 @@ def get_headers():
     admin_api_key = environ.get('API_KEY_PROD').encode()
     auth_token = b64encode(app_id + b":" + admin_api_key).decode().replace('=',
                                                                            '').replace(
+
```

Comment:
Unrelated to the commit, please undo => just delete the commit.

Reply:
✅

Reply:
I don't understand why you are solving this comment. I asked you to undo the change, which in this case consists in deleting the commit.